contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -1,68 +1,704 @@
+          Unit category: NX_ANGLE
+        - **spatial0**: Fast-axis spatial coordinate (or second slow axis if spatially integrated)
+          Unit category: NX_LENGTH
+        - **spatial1**: Slow-axis spatial coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
+          Unit category: NX_LENGTH
+        - **delay**: Calibrated delay time. Unit category: NX_TIME (s).
+        - **polarization_angle**: Linear polarization angle of the incoming or
+          outgoing beam. This could be a link to
+          /entry/instrument/beam/incident_polarization_angle or
+          /entry/instrument/beam/final_polarization_angle if they exist.
+          Unit category: NX_ANGLE (° or rad)
+        - **ellipticity**: Ellipticity of the incoming or outgoing beam.
+          Could be a link to /entry/instrument/beam/incident_ellipticity or
+          /entry/instrument/beam/final_ellipticity if they exist.
+          Unit category: NX_ANGLE (° or rad)
+      \@signal:
+        enumeration: [data]
+      data(NX_NUMBER):
+        unit: NX_ANY
+        doc: |
+          Represents a measure of one- or more-dimensional photoemission counts, where the
+          varied axis may be for example energy, momentum, spatial coordinate, pump-probe
+          delay, spin index, temperature, etc. The axes traces should be linked to the
+          actual encoder position in NXinstrument or calibrated axes in NXprocess.
+      energy(NX_NUMBER):
         doc: |
           Calibrated energy axis.
           
           This could be a link to either
           /entry/process/energy_calibration/calibrated_axis or
           /entry/process/energy_correction/calibrated_axis.
         \@type:
-          type: NX_CHAR
-          doc: |
-            The energy can be either stored as kinetic or as binding energy.
-          enumeration:
-            kinetic:
-              doc:
-              - |
-                Calibrated kinetic energy axis.
-              - |
-                In case the kinetic energy axis is referenced to the Fermi level :math:`E_F`
-                (e.g., in entry/process/energy_referencing), kinetic energies :math:`E` are
-                provided as :math:`E-E_F`.
-              - |
-                xref:
-                  spec: ISO 18115-1:2023
-                  term: 3.35
-                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
-            binding:
-              doc:
-              - |
-                Calibrated binding energy axis.
-              - |
-                xref:
-                  spec: ISO 18115-1:2023
-                  term: 12.16
-                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
-      \@energy_indices:
-        exists: optional
+          exists: recommended
       \@energy_depends:
         type: NX_CHAR
         exists: optional
```

Comment:
similar

Reply:
This is not defined in NXdata_mpes, so we would loose all the description in the docstring if we remove it here. We can also add the description of \@energy_depends to the base class if you prefer.